specs/latest/2.0/index.html

```diff
@@ -3325,13 +3325,16 @@ <h2 class="no-toc">Table of contents</h2>
     </div>
 
     <p>
-        If an ESSL1 fragment shader writes to neither <code>gl_FragColor</code> nor <code>gl_FragData</code>,
-        the values of the fragment colors following shader execution are untouched. If corresponding output
```

Comment:
Why have you removed the ESSL1-related lines? You can still use ESSL1 shaders in OpenGL ES 3 & WebGL 2.

Reply:
It is covered by the third paragraph added to this section. ("Statically used built-in fragment output variables default to zero[...]")